### PR TITLE
Move tests to GH actions

### PR DESCRIPTION
**Pull request by bookernath**
Remove travis, use GH actions for testing instead.

### CODE REVIEW

**Review by matthewvolk — COMMENTED** [review]
🍹 Small nitpick, but we might want to update the README's second sentence from:
```
Tested with python 3.8
```
to 
```
Tested with python 3.7, 3.8, 3.9
```

**Comment by bookernath** [comment]
@matthewvolk updated the readme, thanks.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
```
Comment (bc-chaz) [inline .github/workflows/ci.yml:17]:
```suggestion
    - uses: actions/checkout@v3
```

**Review by bc-chaz — APPROVED** [review]
lgtm

**Pull request merged**